hoot-core-test/src/test/cpp/hoot/core/TestUtils.cpp

```diff
@@ -210,6 +213,8 @@ void TestUtils::resetEnvironment()
   {
     rr->reset();
   }
+  //  Reset the pseudo random number generator seed
+  Tgs::Random::instance()->seed();
```

Comment:
There were a couple of tests that got out of whack with the upgrade to GDAL for no apparent reason.  Come to find out they rely on random numbers that, in some cases, weren't being reset properly.